CHANGELOG.rst

```diff
@@ -53,6 +53,8 @@ Changed
 - When ``plot_histogram()`` or ``plot_state()`` are called from a jupyter
   notebook if there is network connectivity the interactive plots will be used
   by default (#862, #866)
+- Breaking change: ``BaseJob`` API changed, any job constructor must be passed
+  the backend used to run them and a unique job id (#936).
```

Comment:
Can you double-check if some adjustments are needed in the table at `doc/releases.rst`? It doesn't look like this changes will really affect users, but just in case.

Reply:
Done. Nothing to add. I think pointing to the PR is enough.

Reply:
Thank for checking :+1: